### DISCUSSION TITLE
Manage unsafe feature by volume

### DISCUSSION THREAD

**Discussion by realjench** [hardcore-sushi/DroidFS#292] [Ideas]
Hello, great job!

It would be nice to have this feature, for example, a container used for backup application should be exposed to system file provider but not others volumes

Thanks

**Comment by hardcore-sushi** [comment]
Do you mean for all unsafe features? It has already been requested for volume exposing in #240.

Do you have an idea on how to integrate this to the UI?

**Reply by realjench** [reply]
Sorry I didn't see #240 but at first sight I think it was the case, when you long click on a volume you see settings on the three dot, then you see all options.

But even if you use it this way it apply settings to all volumes.

Look like the UI already is ready for this feature.

Thanks for your work

Ah mais vous parlez français   :-) moi je suis vers le lac léman, j'aimerai bien vous envoyer une petite donation mais pas en crypto, paypal?

Merci

**Reply by hardcore-sushi** [reply]
This doesn't solve the problem in cases where the volume is opened but not remembered.

I think only "Allow screenshots", "Keep volume open", and volume exposing features make sense to be volume specific, as the others require actions from the user.

I'd rather implement #240 first as it is more important, and then possibly the two others features if it attracts the interest of other people.
